actions/bind

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/python3
+# -*- mode: python -*-
+#
+# This file is part of Plinth.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+"""
+Configuration helper for BIND server.
+"""
+
+import argparse
+
+from plinth import action_utils
+
+
+CONFIG_FILE = '/etc/bind/named.conf.options'
+
+def parse_arguments():
+    """Return parsed command line arguments as dictionary"""
+    parser = argparse.ArgumentParser()
+    subparsers = parser.add_subparsers(dest='subcommand', help='Sub command')
+
+    configure = subparsers.add_parser('configure', help='Configure Minetest')
```

Comment:
We need to add a parser like this:
```subparsers.add_parser('setup', ...)```

Otherwise, calling the bind action script with 'setup' parameter, won't have the result that we want.